1.8/Dockerfile

```diff
@@ -1,45 +1,42 @@
-FROM centos:centos7
-MAINTAINER Tom Stellard <tstellar@redhat.com>
+FROM centos/s2i-base-centos7
 
-ENV SUMMARY="Red Hat Go Toolset 7 container image" \
-    DESCRIPTION="Platform for building applications using Red Hat Go Toolset 7"
+ENV NAME=golang \
+    VERSION=1.8 \
+    SCL_PATH=/opt/rh/go-toolset-7/
 
-LABEL com.redhat.component="go-toolset-7-docker" \
-      name="devtools/go-toolset-7-rhel7" \
-      version="7" \
-      summary="$SUMMARY" \
+ENV SUMMARY="Platform for building and running Go $VERSION based applications" \
+    DESCRIPTION="Go $VERSION available as docker container is a base platform for \
+building and running various Go $VERSION applications and frameworks. \
+Go is an easy to learn, powerful, statically typed language in the C/C++ \
+tradition with garbage collection, concurrent programming support, and memory safety features."
+
+LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="Go Toolset 7"
-
-RUN yum install -y yum-utils && \
-    yum install -y centos-release-scl && \
+      io.k8s.display-name="Go $VERSION" \
+      io.openshift.tags="builder,golang,golang18,rh-golang18,go" \
+      com.redhat.component="go-toolset-7" \
+      name="centos/golang-1.8-centos7" \
```

Comment:
Other sclorg/ images don't use `.` in version part of the name - so `centos/golang-1.8-centos7`

Reply:
If it is not allow/recommended, will remove.